src/updatedialog.cpp

```diff
@@ -0,0 +1,67 @@
+#include "updatedialog.h"
+#include "ui_updatedialog.h"
+
+#include "lootdialog.h" // for MarkdownPage
+#include <QWebChannel>
+
+using namespace MOBase;
+
+UpdateDialog::UpdateDialog(QWidget* parent) :
+  QDialog(parent, Qt::WindowMaximizeButtonHint | Qt::WindowCloseButtonHint), ui(new Ui::UpdateDialog)
+{
+  // Basic UI stuff
+  ui->setupUi(this);
+  connect(ui->installButton, &QPushButton::pressed, this, [&]{ done(QDialog::Accepted); });
+  connect(ui->cancelButton, &QPushButton::pressed, this, [&]{ done(QDialog::Rejected); });
+
+  // Replace a label with an icon
+  QIcon icon = style()->standardIcon(QStyle::SP_MessageBoxQuestion);
```

Comment:
I think this can be done in the `.ui` file directly.

Reply:
I tried. I failed. I don't know how.